docs/retraced/apis/graphql.md

```diff
@@ -2,15 +2,17 @@
 
 The recommended way to search events using an API is to POST your query and variables to the appropriate Retraced GraphQL endpoint.
 
-| API        | Endpoint                                                                                            |
-| ---------- | --------------------------------------------------------------------------------------------------- |
-| Publisher  | `http://localhost:3000/auditlog/publisher/v1/project/{project_id}/graphql`                          |
-| Admin      | `http://localhost:3000/auditlog/admin/v1/project/{project_id}/environment/{environment_id}/graphql` |
-| Enterprise | `http://localhost:3000/auditlog/enterprise/v1/graphql`                                              |
-| Viewer     | `http://localhost:3000/auditlog/viewer/v1/graphql`                                                  |
+| API              | Endpoint                                                                                            |
+| ---------------- | --------------------------------------------------------------------------------------------------- |
+| Publisher        | `http://localhost:3000/auditlog/publisher/v1/project/{project_id}/graphql`                          |
+| Admin            | `http://localhost:3000/auditlog/admin/v1/project/{project_id}/environment/{environment_id}/graphql` |
+| Enterprise       | `http://localhost:3000/auditlog/enterprise/v1/graphql`                                              |
+| Viewer           | `http://localhost:3000/auditlog/viewer/v1/graphql`                                                  |
+| Viewer Paginated | `http://localhost:3000/auditlog/viewer/v1/graphql/paginated`                                        |
 
 ## Search
 
+This query is used for the non paginated graphql api.
```

Comment:
We shouldn't say non paginated here since the existing search API is also paginated but with cursors.